### PR TITLE
fix: Tx modal token name convention

### LINKED ISSUE #658: fix: tx modal token name convention
**Describe the bug**
The token name on the "from wallet" section should match the token name on the "to vault" section

**To Reproduce**
1. Go to vault details or vault transaction (generally for any curve vault)

**Expected behavior**
The token name convention should match between the wallet and vault deposit

**Screenshots**
image

**Environment Information**

**Additional context**

### PR DESCRIPTION
**Pull request by huggingbot**
## Description

- Updated token name for 'to vault' to use `token.symbol` instead of `displayName`.
- Think using the token symbol would fit better as the display name is already shown in the list.

## Related Issue

- Fixes #658

## Motivation and Context

- Token naming consistency.

## How Has This Been Tested?
- Manually check deposit modal on local environment.

## Screenshots (if appropriate):
deposit

### CODE REVIEW

**Review by FoxTheSin — DISMISSED** [review]

**Review by xgambitox — COMMENTED** [review]
Seems like the same on withdraw (maybe other modals too), worth fixing in same PR if you can take a look.

**Review by karelianpie — APPROVED** [review]
LGTM 🙇

**Review by xgambitox — APPROVED** [review]

**Comment by xgambitox** [comment]
Seems all good to me. Will merge once @FoxTheSin does some QA and approves too :)

**Comment by FoxTheSin** [comment]
sorry to be nit picky but....
image

**Comment by FoxTheSin** [comment]
this should be how it looks it rolls over
image

**Comment by FoxTheSin** [comment]
(side note it's consistent with mobile version*)

**Comment by xgambitox** [comment]
> sorry to be nit picky but.... image
> 
> steth-weth pool is the only one that didn't get the "..."

If we can fix that it does not cut out from the bottom, that would be the best solution imo, since we will probably have more vaults in the future that might have special characters or spacings that make the text only viewable in two separate lines.

Im fine either by fixing here or creating a separate issue for this one.

**Review by karelianpie — APPROVED** [review]

**Review by turtlemoji — APPROVED** [review]

**Pull request merged**
